examples/usecases/ecommerce-session-based-next-item-prediction-for-fashion.ipynb

```diff
@@ -0,0 +1,1546 @@
+{
```

Comment:
<p>I guess better to say <code>We will Tag the columns accordingly based on the type of each column.</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/664/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>